utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelCreator.ts

```diff
@@ -29,7 +29,8 @@ export class TestIModelCreator {
     { description: "Another one", changesetIndexes: [4, 5, 6] }
   ];
 
-  private readonly _iModelDescription = "Some description";
+  private readonly _nonInitializedIModelDescription = "Initializing iModel";
```

Comment:
we can make the text more descriptive - "Reusable iModel entity creation in progress", "Reusable iModel creation completed" or something similar

Reply:
updated